doc/contribute/external.rst

```diff
@@ -1,5 +1,7 @@
 .. _external-contributions:
 
+.. _contributing_external_components:
```

Comment:
why do you need a new anchor here?

Reply:
Oh good catch. I think I was probably looking at reworking the structure of this document because it has multiple H1s and that's not quite right. I have dropped the new target for now, and may provide a later update for the headings situation.